driver-core/src/test/functional/com/mongodb/JsonTestServerVersionChecker.java

```diff
@@ -31,7 +31,7 @@
 import static java.util.Arrays.asList;
 
 public final class JsonTestServerVersionChecker {
-    private static final List<String> TOPOLOGY_TYPES = asList("sharded", "replicaset", "single");
+    private static final List<String> TOPOLOGY_TYPES = asList("sharded", "sharded-replicaset", "replicaset", "single");
```

Comment:
Not doing the check that the shards are actually replica sets, but I'm not sure it matters, as we always start them that way in Evergreen.

Reply:
And as of 3.6 standalones aren't even supported as shards.